starters/vue3-apollo-quasar/.storybook/main.js

```diff
@@ -12,12 +12,15 @@ module.exports = {
   webpackFinal: async (config, { configType }) => {
     // register webpack path aliases
     config.resolve.alias['~storybook'] = path.resolve(__dirname);
-    config.resolve.alias['@'] = path.resolve(__dirname, '..', 'src');
+    config.resolve.alias['src'] = path.resolve(__dirname, '..', 'src');
```

Comment:
keep both to avoid brocken changes in the existing code

Reply:
Thanks